test/OrchardCore.Tests/Apis/Context/SiteContext.cs

```diff
@@ -1,13 +1,35 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
+using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
 using OrchardCore.Apis.GraphQL.Client;
 using OrchardCore.ContentManagement;
+using OrchardCore.Documents;
+using OrchardCore.Environment.Extensions.Features;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Indexing;
+using OrchardCore.Lucene;
+using OrchardCore.Lucene.Model;
+using OrchardCore.Lucene.Services;
+using OrchardCore.Modules;
+using OrchardCore.Recipes.Events;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+using OrchardCore.Settings;
 
 namespace OrchardCore.Tests.Apis.Context
 {
     public class SiteContext : IDisposable
     {
+        private const string luceneRecipePath = "Areas/TheBlogTheme/Recipes";
+        private const string luceneRecipeName = "blog.lucene.recipe.json";
```

Comment:
These are only related to the Blog recipes, so keep them in the Blog-related classes, not in this generic root one (`RunLuceneRecipe`'s name should also reflect this; it could accept a recipe name parameter to make it generally usable).